<launch>>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
        
        <rosparam file="$(find buct_navigation)/param/teb/teb_local_planner_params.yaml" command="load" />
        <rosparam file="$(find buct_navigation)/param/teb/global_planner_params.yaml" command="load" />
        
        <rosparam file="$(find buct_navigation)/param/teb/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find buct_navigation)/param/teb/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find buct_navigation)/param/teb/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find buct_navigation)/param/teb/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find buct_navigation)/param/teb/costmap_converter_params.yaml" command="load" ns="local_costmap" />
        <!--rosparam file="$(find buct_navigation)/param/teb/move_base_params.yaml" command="load" /-->
           
           
            <!--remap from="cmd_vel" to="raw_cmd_vel"/-->
  </node>
  

  
</launch>

   
